cql/src/main/java/io/stargate/cql/CqlActivator.java

```diff
@@ -86,6 +89,19 @@ protected List<ServicePointer<?>> dependencies() {
 
   private static Config makeConfig() {
     try {
+      Config c;
+
+      // This is the same system property as used by persistence backends for custom configuration
+      // using `cassandra.yaml`.
+      String cassandraConfigPath = System.getProperty("stargate.unsafe.cassandra_config_path", "");
```

Comment:
Thinking about this more. It's a bad idea to use this existing system property name because the configuration yaml format and available configuration could be different between CQL and the persistence backend's `cassandra.yaml` format. One example is audit logging, the available options is different between DSE and Cassandra 4.0.